elements/lisk-validator/test/validation.ts

```diff
@@ -599,4 +600,21 @@ describe('validation', () => {
 			});
 		});
 	});
+
+	describe.only('isCsv', () => {
```

Comment:
I think you forgot to remove `only`, all the other tests are skipped.

Reply:
Removed